util/get_mml_dependencies.py

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env python3
+
+import argparse
+import os
+import os.path
+import sys
+import yaml
+
+parser = argparse.ArgumentParser(
+    description="""
+Print the names of the .mss files used by a .mml file (CartoCSS style, YAML
+format) and the name of the .mml file itself as a list of ependencies of a
+Makefile. This script needs to be called from the directory where the Makefile
+is located.""")
+parser.add_argument(
+    "-k",
+    "--key",
+    type=str,
+    default="Stylesheet",
+    help="Name of the YAML key on top level to query for"
+)
+parser.add_argument(
+    "input_file",
+    type=argparse.FileType("br"),
+    help="Input file"
+)
+args = parser.parse_args()
+
+# Check the current working directory is correct.
+if os.path.abspath(os.path.dirname(args.input_file.name)) != os.getcwd():
```

Comment:
Why is this important? Or just cd into the correct directory?

Reply:
I removed it and it works from anywhere now.